mautic/mautic
# enable unlimited send of marketing emails

| Q  | A
| --- | ---
| Bug fix? |  No
| New feature? |  Yes
| Automated tests included? | No
| Related user documentation PR URL | WIP
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? |  No
| Deprecations? |  No

#### Description:

Add the ability to send marketing emails unlimited number of times.
Currently, there is no way to send the same email many times, while controlling the email frequency. This feature enables it.
Transactional emails are sent regardless of frequency, and marketing emails follow the frequency rules, but can only be sent once per lead.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Enable unlimited marketing emails s in the email configuration section
3. Send a marketing email to lead, and then send it again and make sure both or sent
